### DISCUSSION TITLE
Support for Thermo Electric Actuators

### DISCUSSION THREAD

**Discussion by rosiaantje** [nliaudat/esp32_8ch_motor_shield#71] [Ideas]
I have a bunch of EMO TM (thermo electric) actuators. I would be very nice if they could be controlled by this board!

**Comment by nliaudat** [comment]
EMO TA (thermo electric) are 24 VAC/VDC or 230 VAC ±15%. 
You can use a simple relay board. 
There are plenty running esphome.

**Reply by rosiaantje** [reply]
Nope this one needs control signal 0-10v or 2-10v.. it's the proportional instead of open closed

**Reply by rosiaantje** [reply]
The one I mean is:

https://assets.imi-hydronic.com/Documents/Catalogues/Netherlands/PDF_low/EMO-TM_NL_low.pdf


I see I was writing it not correct in start post. Should be TM instead of TA. (Corrected)

**Comment by nliaudat** [comment]
Hi, 
The new motor driver TC118s supports 2.4-7.2V - 1500mA. 
It will burn at 10V. 
The second problem is you need to have a huge 24V AC power supply (7W for each valve) + a 24V AC to 10V DC converter (which is hard to find in the market)
